cinema_service/settings.py

```diff
@@ -86,12 +91,15 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ["POSTGRES_DB"],
+        "USER": os.environ["POSTGRES_USER"],
+        "PASSWORD": os.environ["POSTGRES_PASSWORD"],
+        "HOST": os.environ["POSTGRES_HOST"],
+        "PORT": os.environ["POSTGRES_PORT"],
```

Comment:
Code quality issue: It's good that you're using environment variables to manage your database settings. However, it's recommended to provide a default value in case the environment variable isn't set to prevent your application from crashing. You can use `os.getenv('POSTGRES_DB', default='default')` to get an environment variable and provide a default value.